index.html

```diff
@@ -1,13 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <div class="container"></div>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Vanilla Todo</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="container">
+    <h2> 대기중(
+      <span id="pendingTasksNumber">0</span>
```

Comment:
HTML 클래스와 아이디 네이밍에는 lowerCamelCase 보다는 하이픈(-)을 기본적으로 사용한답니다~ 다른것들도 통일해주시면 좋을 것 같아요!
```suggestion
      <span id="pending-tasks-number">0</span>
```

Reply:
HTML 네이밍은 하이픈(-)사용! 잘 몰랐던 내용이었네요 감사합니다!

Reply:
저도 배우고 갑니다